dist/drupal-conventions.yml

```diff
@@ -13,3 +13,32 @@ parameters:
     - module
     - inc
     - theme
+
+  tasks:
+
+    # PHP Code Sniffer.
+    phpcs:
+      standard: '%tasks.phpcs.standard%'
+      ignore_patterns: '%tasks.phpcs.ignore_patterns%'
+      triggered_by: '%tasks.phpcs.triggered_by%'
+      whitelist_patterns: '%tasks.phpcs.whitelist_patterns%'
+      warning_severity: '%tasks.phpcs.warning_severity%'
+      exclude:
+        - 'Drupal.Commenting.Deprecated'
+      metadata:
+        priority: 300
+
+      # PHP Mess Detector.
+    phpmd:
+      exclude: '%tasks.phpmd.exclude%'
+      ruleset: '%tasks.phpmd.ruleset%'
+      triggered_by: '%tasks.phpmd.triggered_by%'
+
+    # Commit message format.
+    git_commit_message:
+      matchers: '%tasks.git_commit_message.matchers%'
+      case_insensitive: false
+      enforce_no_subject_trailing_period: false
+      multiline: false
+      max_body_width: 0
+      max_subject_width: 0
```

Comment:
Maybe we move the `tasks:` block from `base-conventions` to `library-conventions`? I wondering that, since we have to duplicate it, it's better to have it in its final place.

Reply:
Done.